dashboard-widgets.el

```diff
@@ -740,7 +758,7 @@ Argument IMAGE-PATH path to the image."
       (when graphic-mode (insert "\n"))
       ;; If specified, insert a text banner.
       (when-let ((txt (plist-get banner :text)))
-        (if (eq dashboard-startup-banner 'ascii)
+        (if (length= banner 2)
```

Comment:
This looks weird. Try avoid the magic number here. 🤔

Reply:
> This looks weird. Try avoid the magic number here. 🤔

That was the only way i found to check if banner is a cons cell, using listp or consp conflicts with random banner list.

Reply:
If you know a way to determine a cons cell, i'd really appreciate if you could tell me.

Reply:
Fixed, only determining if it is a list fixs this issue.